packages/react-radio/src/components/Radio/renderRadio.tsx

```diff
@@ -10,14 +10,9 @@ export const renderRadio_unstable = (state: RadioState) => {
 
   return (
     <slots.root {...slotProps.root}>
-      <div className={state.containerClassName}>
-        <slots.indicator {...slotProps.indicator} />
-        <slots.input {...slotProps.input} />
-      </div>
-      <slots.label {...slotProps.label}>
-        {state.label.children}
-        {state.subtext && slots.subtext && <slots.subtext {...slotProps.subtext} />}
-      </slots.label>
+      <slots.input {...slotProps.input} />
+      {slots.indicator && <slots.indicator {...slotProps.indicator} />}
```

Comment:
What's the case for the `indicator` slot being null?

Reply:
All of our slots are nullable by default. Although I can't think of any reason someone would set it to null... I'll make it non nullable.

As an aside, I think it'd be better to change all slots to be non-nullable by default, and explicitly include `| null` in the type if they should be nullable. But that's a bigger topic and doesn't need to be addressed right now :)